<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowClass</class>
 <widget class="QMainWindow" name="WindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="departments_volunteers_list">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>441</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="description_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>871</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="unassigned_volunteers_list">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>391</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_line_edit">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>290</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line_edit">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>340</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="interests_line_edit">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>390</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nume</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLabel" name="interests_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>interese</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_volunteer_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>440</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>add volunteer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Window.qrc"/>
 </resources>
 <connections/>
</ui>
